source/hardware/controllers/numark_ns6ii.rst

```diff
@@ -0,0 +1,543 @@
+avoids sudden jumps). The closer the slider is to its neutral position, the
+brighter the middle LED is. The closer the physical slider position is to the
+software position, the dimmer they get. This allows you to judge how far you
+have to move the slider until you have synchronized the fader and how far you
+have to move it until it is at neutral speed. These leds (with the exceptions of
+the middle LED are also repeated on the Jogwheel LCD Display).
+
+
+Mixer Section
+^^^^^^^^^^^^^
+
+The NS6II features 4 identical (with respect to symmetry) Mixer Channels. See :ref:`interface-eq-gain` for a complete explanation of the individual concepts
+
+.. image:: ../../_static/controllers/numark_ns6ii_schematics/mixer.svg
+   :align: left
+   :width: 20%
+
+
+========  ==============================================================  ==========================================
+No.       Control                                                         Function
+========  ==============================================================  ==========================================
+7         :hwlabel:`GAIN`                                                 Channel Gain (see :ref:`interface-eq-gain`)
+8         Channel VuMeter                                                 loudness of signal of that specific channel (after EQ, before Filter and Faders). A white LED indicates a clipping signal (see :term:`level meter`).
+9         :hwlabel:`HIGH`                                                 Channel High/Treble EQ
+10        :hwlabel:`MID`                                                  Channel Mid EQ
+11        :hwlabel:`LOW`                                                  Channel Low/Bass EQ
+12        Channel Fader                                                   Decrease the volume of the channel
+13        :hwlabel:`🎧` (pfl/cue) button                                  Toggle :term:`PFL`. Notice: due to hardware constraints, this always works in numark mode: clicking a single button solos the corresponding channel, to listen to multiple channels, press and then release multiple buttons at once.
+14        :hwlabel:`A OFF B` switch                                       The :term:`crossfader` orientation switches determine to which side of the crossfader the deck is assigned. It can be set to the left or right side, or to the center where it is unaffected by the crossfader knob.
+47        :hwlabel:`FILTER`                                               This knob lets you control the Meta knob of the deck’s Quick Effect.
+38        :hwlabel:`LOAD`                                                 Load the currently selected track from the library into the deck corresponding to the channel
+48        :hwlabel:`A` / :hwlabel:`B`                                     Route Channel to left/right :ref:`Effect Unit <interface-effects>`.
+========  ==============================================================  ==========================================
```

Comment:
The odd numbering from the SVG is confusing already :zany_face:  (I presume that's from the off. manual?)
so how about we simply adopt it so the order in SVG and table match?

Reply:
Btw 48 --> 46

Reply:
yeah, its from the original manual and I agree its suboptimal. I don't understand your proposal however. Can you elaborate?

Reply:
Let's reorder the tabe rows so the order (top -> bottom) matches the order of the numbers in the SVG.

And 48 is probably referring to 46 in the SVG.

Reply:
So just order ascending by `No.`?

Reply:
Nope, simply same order as in th SVG

|| SVG | table |
|-|-|-|
|<td rowspan="11">![image](https://github.com/user-attachments/assets/10fcdbdd-0ce3-4f3e-aaba-23bae24771e9)</td> | 38
|| 46
|| 7
|| 9
|| 8
|| 10
|| 11
|| 47
|| 13
|| 12

Reply:
Ah I see. Sure.

Reply:
fixed. Thank you.